.github/workflows/playwright.yml

```diff
@@ -0,0 +1,59 @@
+name: Playwright Tests
+
+on:
+  pull_request:
+
+jobs:
+  quickstart-test:
+    timeout-minutes: 30
+    name: build-quickstart-contract
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Can we add a space between each step normally we group by `setup`, `install`, `checks`....